.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Cypress Tests
+
+on: 
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  cypress-run-vue2:
+    runs-on: ubuntu-latest
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+      CYPRESS_baseUrl: http://localhost:8000/
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          build: yarn run build
+          start: yarn run serve
+          wait-on: http://localhost:8000/
+          config-file: cypress.json
+          spec: "cypress/integration/**/*"
+          working-directory: andv-front-end
+  
+  # cypress-run-vue3:
```

Comment:
https://github.com/FunctionStream/function-stream/issues/79 this issue has been fixed. I think we can enable `cypress-run-vue3` now.